packages/encryption/src/image-steganography/presets.ts

```diff
@@ -59,9 +59,20 @@ const Preset2022: Preset = {
     options: libV2AlgrDefaults,
 }
 
+const Preset2023: Preset = {
+    type: 'string',
+    description: 'the preset we used for payload V38',
+    width: 1200,
+    // TODO: use a different height with Preset 2021 and Preset 2022
+    height: 681,
+    mask: null,
+    options: libV2AlgrDefaults,
+}
+
 const dimensionPreset: readonly Preset[] = [
-    Preset2021,
+    Preset2023,
     Preset2022,
+    // Preset2021,
```

Comment:
do not comment out the old preset otherwise, we'll unable to decrypt the old payload.